apps/remix-ide/src/blockchain/providers/injected.js

```diff
@@ -16,17 +16,13 @@ class InjectedProvider {
     })
   }
 
-  resetEnvironment () {
+  async resetEnvironment () {
```

Comment:
This function does nothing. Should it still be kept?

Reply:
yeah so it complies with the interface